CRM/Activity/DAO/Activity.php

```diff
@@ -234,6 +234,13 @@ static function &fields() {
           'dataPattern' => '',
           'export' => true,
         ) ,
+        'activity_free_text_search' => array(
```

Comment:
Hi @seamuslee001 Sorry for the confusion, just made the adjustments. please check now.